optional-extra-practice/src/ScoopOption.tsx

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+import Col from 'react-bootstrap/Col';
+import From from 'react-bootstrap/Form';
+import Row from 'react-bootstrap/Row';
+import './App.css';
+function ScoopOption({ name, imagePath, updateItemCount}) {
+
+  //const [order]
+
+  return (
+    <div className="ScoopOption">
+    </div>
+  ); 
+
+
+}
+
+export default ScoopOption;
```

Comment:
Please remove all the unnecessary empty lines, you can still make your code readable by making single empty lines, but not multiple lines